api/api/spec/spec.yaml

```diff
@@ -5752,31 +5773,29 @@ paths:
           $ref: '#/components/responses/ResponseError'
         '401':
           $ref: '#/components/responses/UnauthorizedResponse'
-        '403':
```

Comment:
Why is 403 removed?

Reply:
Don't know, probably an error when merging